### PR TITLE
Add composer.json

### PR DESCRIPTION
**Pull request by blankse**
Can you add a composer.json, so that I can add the extension per composer?

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        {
+            "name": "eZ Publish dev-team & eZ Community",
+            "homepage": "https://github.com/ezsystems/ezxmlinstaller/contributors"
+        }
+    ],
+    "minimum-stability": "dev",
+    "require": {
+        "ezsystems/ezpublish-legacy-installer": "*"
+    }
+}
```
Comment (andrerom) [inline composer.json]: newline

**Comment by andrerom** [comment]
+1

**Comment by blankse** [comment]
Added legacy extension name and the newline at the end.

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "description": "The eZ XML Installer extension is a plattform to define proccesses for eZ Publish and execute them.",
+    "type": "ezpublish-legacy-extension",
+    "license": "GPL-2.0",
+    "authors": [
+        {
+            "name": "eZ Publish dev-team & eZ Community",
+            "homepage": "https://github.com/ezsystems/ezxmlinstaller/contributors"
+        }
+    ],
+    "minimum-stability": "dev",
```
Comment (lolautruche) [inline composer.json]: This is not needed.

**Comment by lolautruche** [comment]
+1

**Comment by blankse** [comment]
Removed minimum-stability

**Comment by lolautruche** [comment]
Thanks @blankse !

For next time, try not to do PRs based on your master branch. This might save your life :wink:

**Comment by blankse** [comment]
Ok @lolautruche, I will try it :)

**Pull request merged**